development/mimir-microservices-mode/dev.dockerfile

```
@@ -1,4 +1,6 @@
-FROM golang:1.20.5
+
+ARG BUILD_IMAGE=use-compose-up-to-build-image
+FROM ${BUILD_IMAGE}
```

The braces aren't necessary.

```suggestion
FROM $BUILD_IMAGE
```